css/properties/mask.json

```diff
@@ -27,11 +27,11 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": true,
+              "version_added": "2",
               "notes": "From Firefox 10, the default color space when handling masks is sRGB. Previously, the default and only supported color space was linear RGB. This changes the appearance of mask effects, but brings Firefox into compliance with the second edition of the SVG 1.1 specification."
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "4"
```

Comment:
I'm assuming this is based on the value for Firefox. If so we should pull down the note, or substantiate that the behavior is different for Android.